publify_core/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,6 +1,6 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
+  <div class="sidebar-body">
```

Comment:
Why is this here? 

You need to create a new branch for each new PR. Otherwise, you will be including older code in newer PRs, which means I am reviewing code for unrelated issues.